draft-ietf-tls-keylogfile.md

```diff
@@ -340,6 +386,28 @@ Change controller:
 : IESG
 {: spacing="compact"}
 
+## SSLKEYLOGFILE labels registry
+
+IANA is requested to create a new registry "SSLKEYLOGFILE Labels", within the
+existing "Transport Layer Security (TLS) Parameters" registry page.
+This new registry reserves labels used for SSLKEYLOGFILE entries.
+The initial contents of this registry are as follows.
+
+| Value | Description | Reference |
+| --- | --- | --- |
+| CLIENT_RANDOM | Master secret in TLS 1.2 and earlier | This document |
+| CLIENT_EARLY_TRAFFIC_SECRET | Secret for client early data records | This document |
+| EARLY_EXPORTER_MASTER_SECRET | Early exporters secret | This document |
+| CLIENT_HANDSHAKE_TRAFFIC_SECRET | Secret protecting client handshake | This document |
+| SERVER_HANDSHAKE_TRAFFIC_SECRET | Secret protecting server handshake | This document |
+| CLIENT_TRAFFIC_SECRET_0 | Secret protecting client records post handshake | This document |
+| SERVER_TRAFFIC_SECRET_0 | Secret protecting server records post handshake | This document |
+| EXPORTER_SECRET | Exporter secret after handshake | This document |
+| ECH_SECRET | HPKE KEM shared secret used in the ECH | This document |
+| ECH_CONFIG | ECHConfig used for construction of the ECH | This document |
+
+New assignments in the "SSLKEYLOGFILE Labels" registry will be administered by IANA through
+IETF Review procedure {{!RFC8126}}.
```

Comment:
I'm going to start a thread on this.  I think that we should instead go for Specification Required, with experts advised to reject registrations for secrets that can be used to derive other secrets (other than _TRAFFIC_SECRET_0, that should be unnecessary).